wscript

```diff
@@ -60,6 +60,7 @@ def build(bld):
         bld.recurse('test')
 
     bld.recurse('src/tables')
+    bld.recurse('examples/creating_printing_table')
```

Comment:
This line should be after `bld.recurse('test')` in the top-level clause.
If tables is used as a dependency, then the example should *not* be defined (similarly to the test).